ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -566,115 +554,79 @@ def get_disk_safety(logger):
                     if policy[1] > max_lost_disks:
                         max_lost_disks = policy[1]
 
-            disk_lost_overview = {}
-            total_objects = 0
+            lost_disks = {}
+            details = {'total_objects': 0, 'lost_disks': lost_disks}
+            disk_lost_overview[alba_backend.name] = details
 
             for namespace in namespaces:
                 for bucket_safety in namespace['bucket_safety']:
                     bucket = bucket_safety['bucket']
-                    objects = bucket_safety['count']
-                    total_objects += objects
-                    applicable_dead_osds = bucket_safety['applicable_dead_osds']
+                    disk_lost = abs(bucket[2] - bucket_safety['applicable_dead_osds'] - (bucket[0] + bucket[1]))
+                    remaining_safety = bucket_safety['remaining_safety']
+                    if disk_lost not in lost_disks:
+                        lost_disks[disk_lost] = {"remaining_safety": remaining_safety, "objects_to_repair": 0}
                     # Amount of lost disks at this point
-                    bucket[2] = bucket[2] - applicable_dead_osds
-                    backend_overview = bucket[2] - (bucket[0] + bucket[1])
-                    if backend_overview not in disk_lost_overview:
-                        disk_lost_overview[backend_overview] = 0
-                    disk_lost_overview[backend_overview] += objects
-            for backend_overview, objects in disk_lost_overview.iteritems():
-                lost = {
-                    'measurement': 'disk_lost',
-                    'tags': {
-                        'backend_name': alba_backend.name,
-                        'disk_lost': backend_overview
-                    },
-                    'fields': {
-                        'total_objects': total_objects,
-                        'objects': objects
-                    }
+                        lost_disks[disk_lost]['objects_to_repair'] += bucket_safety['count']
```

Comment:
This line is obsolete, since you will only enter this ```if``` statement once, the ```objects_to_repair``` can be put directly in the line above